python_modules/dagster/dagster/_config/pythonic_config/__init__.py

```diff
@@ -558,18 +571,39 @@ def __init__(self, **data: Any):
             k: v for k, v in self._as_config_dict().items() if k in data_without_resources
         }
         resolved_config_dict = config_dictionary_from_values(casted_data_without_resources, schema)
-        self._config_schema = _curry_config_schema(schema, resolved_config_dict)
 
-        # We keep track of any resources we depend on which are not fully configured
-        # so that we can retrieve them at runtime
-        self._nested_partial_resources: Mapping[str, CoercibleToResource] = {
-            k: v for k, v in resource_pointers.items() if (not _is_fully_configured(v))
-        }
+        self._state__internal__ = ConfigurableResourceFactoryState(
```

Comment:
this is pretty nice, is the goal down the line to remove the `@property` impls and reference everything through the state? or keep the property access pattern for cleanliness

Reply:
We should probably get rid of them to minimize namespace pollution with user-defined properties, but I don't think it is critical.

Reply:
I don't have a super intentional goal here. This in my mind was the most expedient way to bang this out.